extensions/http/src/test/java/org/mule/extension/http/policy/HttpListenerPolicyPointcutParametersFactoryTestCase.java

```diff
@@ -43,7 +42,7 @@ public void supportsHttpListener() {
   @Test
   public void doesNotSupportHttpRequester() {
     assertThat(factory
-        .supportsSourceIdentifier(new Builder().withNamespace("http").withName("request").build()),
+        .supportsSourceIdentifier(ComponentIdentifier.builder().withNamespace("http").withName("request").build()),
```

Comment:
I can't use it because there's already an static import for another buider() function.